app/scripts/modules/core/src/managed/ManagedResourceStatusIndicator.tsx

```diff
@@ -177,30 +185,125 @@ const logClick = (label: string, resourceId: string, status: ManagedResourceStat
+        </p>
+      `;
+  }
+  if (regions.length > 1) {
+    body += `
+        <p>
+          <div class="horizontal top sp-padding-m alert alert-warning">
+            <i class="fa fa-exclamation-triangle sp-margin-m-right sp-margin-xs-top"></i>
+            <span>${isPaused ? 'Resuming' : 'Pausing'} management of this resource will ${isPaused ? 'resume' : 'pause'}
+             management of this resource in the following managed regions: <b>${regions.join(', ')}</b>.
+            </span>
+          </div>
+        </p>`;
+  }
+  return body;
+};
+
+const LearnMoreLink = ({ resourceSummary }: { resourceSummary: IManagedResourceSummary }) => (
+  <a
+    target="_blank"
+    onClick={() => logClick('Status docs link', resourceSummary.id, resourceSummary.status)}
+    href={`https://www.spinnaker.io/reference/managed-delivery/resource-status/#${resourceSummary.status.toLowerCase()}`}
+  >
+    Learn more
+  </a>
+);
+
+const PopoverLinks = ({
+  resourceSummary,
+  application,
+  hidePopover,
+}: {
+  resourceSummary: IManagedResourceSummary;
+  application: Application;
+  hidePopover: () => void;
+}) => (
+  <div className="horizontal right">
+    {!resourceSummary.isPaused && (
+      <p className="sp-margin-m-top sp-margin-xs-bottom">
+        <button className="passive" onClick={() => toggleResourcePause(resourceSummary, application, hidePopover)}>
+          <i className="fa fa-pause" /> Pause management of this resource
+        </button>
+      </p>
+    )}
+    {resourceSummary.isPaused && !application.isManagementPaused && (
+      <p className="sp-margin-m-top sp-margin-xs-bottom">
+        <button className="passive" onClick={() => toggleResourcePause(resourceSummary, application, hidePopover)}>
+          <i className="fa fa-play" /> Resume management of this resource
+        </button>
+      </p>
+    )}
+    {application.isManagementPaused && (
+      <p>
+        <UISref to="home.applications.application.config" params={{ section: 'managed-resources' }}>
+          <a>Resume application management</a>
+        </UISref>
+      </p>
+    )}
+  </div>
 );
 
 export interface IManagedResourceStatusIndicatorProps {
   shape: 'square' | 'circle';
   resourceSummary: IManagedResourceSummary;
+  application: Application;
 }
 
 export const ManagedResourceStatusIndicator = ({
   shape,
-  resourceSummary: { id, status },
+  resourceSummary,
+  application,
 }: IManagedResourceStatusIndicatorProps) => {
+  const { status } = resourceSummary;
+  const PopoverContents = ({ hidePopover }: IHoverablePopoverContentsProps) => (
+    <>
+      {viewConfigurationByStatus[status].popoverContents(resourceSummary, application)}
+      <PopoverLinks resourceSummary={resourceSummary} application={application} hidePopover={hidePopover} />
+    </>
+  );
   return (
     <div className="flex-container-h stretch ManagedResourceStatusIndicator">
-      <HoverablePopover template={viewConfigurationByStatus[status].popoverContents(id)} placement="left">
```

Comment:
is the reason for switching to `Component` purely that you need access to the`hidePopover` callback?

Reply:
It is only for the `hidePopover`, which we need because the popover has a higher z-index than the modal. We do the same thing with the `<NotificationsPopover>`, and I think that's why the HoverablePopover takes a Component.